ruby/2.1.5/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM socrata/base
+MAINTAINER Socrata <sysadmin@socrata.com>
+
+RUN DEBIAN_FRONTEND=noninteractive apt-get -y update && \
+  DEBIAN_FRONTEND=noninteractive apt-get --force-yes -fuy install software-properties-common && \
+  DEBIAN_FRONTEND=noninteractive add-apt-repository -y ppa:brightbox/ruby-ng && \
+  DEBIAN_FRONTEND=noninteractive apt-get -y update && \
+  DEBIAN_FRONTEND=noninteractive apt-get install -y ruby2.1 && \
```

Comment:
So you are installing the 2.1 package but calling it 2.1.5.  Does that make sense or should we just call it 2.1?  Won't the package potentially install 2.1.6, etc. at some point?  This reminds me of the ubuntu situation we got into where "1.9.1" was actually "1.9.3" or some such.

(not trying to be nitpicky...)